### DISCUSSION TITLE
How to customize daisyui color varibles?

### DISCUSSION THREAD

**Discussion by Reverier-Xu** [saadeghi/daisyui#2821] [Q&A] [answered]
Hi there! I want to custom the button background in daisyui:

```scss
.btn {
    @apply bg-base-content/5 backdrop-blur  ...;
}
```

but the color modifier classes such as `btn-primary` do not work properly, the `.btn` class has a higher priority.

I noticed that `btn-primary` just changes the css variable `--btn-color`, so the proper solution is setup the default value of `--btn-color`. is there any way to do this? it seems that items in `tailwind.config.js` cannot refer to other variables.

**Comment by saadeghi** [comment, marked as answer]
What is your expected behavior?  
Because I think it makes sense when you set a background color for `btn` it effects everything with `btn` class.  

I would suggest defining the colors in [theme config](https://daisyui.com/docs/themes/#-4) instead of manually customizing the color of a button.

Or you can use a new class name like `blurry-btn` or something instead of rewriting the style for `btn` if you want a special style of button.

Let me know if you have a question.
